app/scripts/modules/oracle/serverGroup/details/resize/resizeServerGroup.html

```diff
@@ -0,0 +1,33 @@
+<div modal-page class="confirmation-modal">
+  <task-monitor monitor="taskMonitor"></task-monitor>
+  <form role="form" ng-if="!ctrl.submitting">
+    <modal-close dismiss="$dismiss()"></modal-close>
+    <div class="modal-header">
+      <h3>Resize {{serverGroup.name}}</h3>
+    </div>
+    <div class="modal-body confirmation-modal">
+      <div class="form-horizontal">
+        <oracle-bmcs-resize-capacity
+                             server-group="serverGroup"
+                             form-methods="formMethods"
+                             application="application"
+                             command="command"></oracle-bmcs-resize-capacity>
+      </div>
+      <div class="row" ng-if="command.platformHealthOnlyShowOverride">
+        <div class="col-sm-10 col-sm-offset-1">
+          <platform-health-override command="command"
+                                    platform-health-type="'Google'"
+                                    show-help-details="true">
+          </platform-health-override>
+        </div>
+      </div>
+      <task-reason command="command"></task-reason>
+
+    </div>
+    <gce-footer action="ctrl.resize()"
```

Comment:
You're welcome to use `gce-footer`, but I noticed an Oracle footer directive above - just checking that this is right.